vivisect/parsers/elf.py

```diff
@@ -613,7 +610,10 @@ def applyRelocs(elf, vw, addbase=False, baseaddr=0):
 
             if arch in ('arm', 'thumb', 'thumb16'):
                 if rtype == Elf.R_ARM_JUMP_SLOT:
-                    symidx = Elf.getRelocSymTabIndex(r.r_info)
+                    symidx = r.getSymTabIndex()
```

Comment:
use the reloc's method getSymTabIndex()

Reply:
![yes-this-pleases-gaga](https://user-images.githubusercontent.com/45212823/70334353-4e942b80-1813-11ea-9aaa-76e601192308.jpg)

Reply:
:D